CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+Contributing
+---------------------------
+
+# Contributing
+
+## Workflow
+
+MongoDB welcomes community contributions! If you’re interested in making a contribution to  Terraform MongoDB Atlas provider, please follow the steps below before you start writing any code:
+
+1. Sign the [contributor's agreement](http://www.mongodb.com/contributor). This will allow us to review and accept contributions.
```

Comment:
Instead of `1.` all the way down the list, should we either increment the numbered list or switch over to a bulleted list?